background_scripts/main.coffee

```diff
@@ -339,6 +339,17 @@ updateOpenTabs = (tab) ->
 setBrowserActionIcon = (tabId,path) ->
   chrome.browserAction.setIcon({ tabId: tabId, path: path })
 
+# This color should match the blue of the Vimium browser popup (although it looks a little darker, to me?).
+chrome.browserAction.setBadgeBackgroundColor {color: [102, 176, 226, 255]}
```

Comment:
Elsewhere we have spaces around braces and the keys inside of them. Let's use spaces for consistency.